Monika After Story/game/script-moods.rpy

```diff
@@ -882,3 +882,26 @@ label mas_mood_excited:
             m 3eua "I know I'm always excited to see you every day."
             m 1hub "Either way, I'm glad that you're happy!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_mood_database,
+            eventlabel="mas_mood_burnt",
```

Comment:
eventlabel should properly encompass the content. If the mood is `burnt out`, we should keep the full name here.
```suggestion
            eventlabel="mas_mood_burnt_out",
```